lib/relay.js

```diff
@@ -112,6 +114,9 @@ function responseHandler(filterRules, config) {
         headers: headers,
         method,
         body,
+        agentOptions: {
+          ca: (() => {return config.caCert ? fs.readFileSync(path.resolve(process.cwd(), config.caCert)) : undefined;})(),
```

Comment:
Good idea, missed that this will get reevaluated on request, I'll push an update.